bin/cli.js

```diff
@@ -2,7 +2,7 @@
 const cli = require('../dist/cli')
 const index = require('../dist/index')
 
-var argv = require('yargs')
+var argv = require('yargs')()
```

Comment:
Interesting, doesn't the test cover this validation?

Reply:
https://github.com/Azure/avocado/issues/200